### PR TITLE
Added more output to the logging

### PR DESCRIPTION
**Pull request by sophieball**
1. Added more output to the log, so we could use them to debug.
2. Added more functions in `src/convo_politeness.py` for training a politeness classifier. 
3. I haven't added a test for training politeness classifier - loading `src/convo_politeness.py` requires downloading nltk data but I haven't figure out how to do it with travis. And the rest of the program (apart from training), is just `receive_data`, which has its own test.

### CODE REVIEW

File: src/convo_politeness.py [hunk 1]
```diff
@@ -46,18 +69,106 @@ def get_politeness_score(comments):
...
+# split data to do cross validation
+def cross_validate(comments):
+  corpus = prepare_corpus(comments)
+  corpus = transform_politeness(corpus)
+  scores = polite_score(corpus, need_stanford=False)
+  y = scores["label"]
+  X = pick_features(scores)
+  X_train, X_test, y_train, y_test = model_selection.train_test_split(
+      X, y, test_size=test_size, random_state=0)
+  #clf = linear_model.LogisticRegression(random_state=0).fit(X_train, y_train)
```
Comment (CaptainEmerson) [inline src/convo_politeness.py:134]: rm, and below?

File: src/create_features.py [hunk 2]
```diff
@@ -126,7 +126,7 @@ def create_features(comments_df, training):
 
   # get politeness scores for all comments
   all_stanford_polite = convo_politeness.get_politeness_score(comments_df)
-  comments_df = comments_df.join(all_stanford_polite.set_index("_id"), on="_id")
+  #comments_df = comments_df.join(all_stanford_polite.set_index("_id"), on="_id")
```
Comment (CaptainEmerson) [inline src/create_features.py:129]: em?

Comment (sophieball) [reply src/create_features.py]: emoved

File: src/convo_politeness.py (hunk 1 above)
Comment (sophieball) [reply src/convo_politeness.py]: removed

**Pull request merged**